yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -374,7 +374,39 @@ Internet-Draft                 YANG Semver                  October 2022
    modifier strings.  E.g., artifact version "1.2.3_non_compatible" MUST
    NOT be defined if artifact version "1.2.3" has already been defined.
 
-3.3.1.  YANG Semver with submodules
+3.3.1.  Branching Limitations with YANG Semver
+
+   YANG artifacts that use the YANG Semver revision-label scheme MUST
+   ensure that two artifacts with the same MAJOR version number and no
+   _compatible or _non_compatible modifiers are backwards compatible.
+   Therefore, certain branching schemes cannot be used with YANG Semver.
+   For example, the following branching scheme cannot use YANG Semver
+   for its revision-label scheme:
+
```

Comment:
Maybe replace that last sentence with this ?  For example, the following branched parent-child module relationship using the following YANG Semver revision labels is not supported: